io.github.atari800.Atari800.yml

```diff
@@ -0,0 +1,54 @@
+app-id: io.github.atari800.Atari800
+runtime: org.kde.Platform
+runtime-version: '6.4'
+sdk: org.kde.Sdk
+command: atari800
+build-options:
+  no-debuginfo: true
+  strip: true
+modules:
+  - name: libsdl
+    sources:
+      - type: archive
+        url: https://libsdl.org/release/SDL-1.2.15.tar.gz
+        sha256: d6d316a793e5e348155f0dd93b979798933fb98aa1edebcc108829d6474aad00
+      - type: patch
+        path: patch/sdlpatch.txt
+        strip-components: 1
+  - name: atari800
+    config-opts:
+      - --with-video=sdl
+      - --with-sound=sdl
+      - --enable-pngcodec
+      - --with-mp3=lame
+    post-install:
+      - install -Dm 0644 ${FLATPAK_ID}.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
```

Comment:
this file is missing, but again it is upstream https://github.com/atari800/atari800/blob/master/debian/atari800.desktop

Reply:
Instructions said to only commit the manifest.  Will commit the rest.

Reply:
I mentionned it's in the upstream repo. you should use that one instead.

Reply:
When I try to use that file, I get a warning about the icon name not matching the flatpak app id.  Another comment mentioned using desktop-file-edit to modify the .desktop file entries.  I cannot find desktop-file-edit documented anywhere.

Reply:
What you're telling me to do breaks the build.  If I cannot update that .desktop file with the correct icon name, this isn't going to work.

Reply:
No. I'm telling use the one that ship with upstream. If it needs fixing because it's broken upstream it can be fixed.

`man desktop-file-edit` for the documentation.

Reply:
I used sed to edit the file to put in the correct icon name.

Reply:
please don't use sed for that. that's what `destkop-file-edit` is for.

Reply:
Updated to use desktop-file-edit.